README.md

```diff
@@ -37,26 +46,7 @@ Operators can behave like managed service providers. Their user interface on the
 
 ## Getting Started 
 
-### Installation
-
-Install OLM on a Kubernetes or OpenShift cluster by following the [installation guide].
```

Comment:
Since a lot of what's in the installation guide, i.e "Installing a released version", "Subscribing to a package and channel" etc can be categorized as components of the user facing docs, we can probably simplify things here by deleting `install.md`(as you currently have done in this PR), and replacing this with 
```Prerequisite: running Minikube/Kind cluster 
$ git clone repo locally
$ make run-local ```

Reply:
I removed the OpenShift from here and revert the installation guide since my goal was more only just to add the olm docs here. So, we can get merged this one without requiring to move or re-check what is in the installation guide and how it should be. I hope that is fine.